packages/esm-patient-labs-app/src/test-results/grouped-timeline/grouped-timeline.styles.scss

```diff
@@ -64,6 +65,11 @@
     @include type.type-style('body-compact-01');
     color: $text-02;
   }
+
+  &:hover {
+    background-color: colors.$blue-10;
+    border-top: 2px colors.$blue-30;
+  }
 }
```

Comment:
These changes effect the same styling mods made to the Timeline view to the GroupedTimeline view (what you see after selecting a panel in the tree view):

![CleanShot 2024-07-31 at 1  03 01@2x](https://github.com/user-attachments/assets/be579e1f-2411-4118-ae54-ec3f5ac6bf36)